packages/blocks/src/root-block/widgets/scroll-anchoring/index.ts

```diff
@@ -0,0 +1,234 @@
+import type { DocMode } from '@blocksuite/affine-model';
```

Comment:
Put the new widget in `affine/widget-scroll-anchoring` as a new package, and import it in `blocks`